modern/src/reports/ReportFilter.js

```diff
@@ -56,56 +57,80 @@ const ReportFilter = ({ children, handleSubmit, showOnly }) => {
   }
 
   return (
-    <>
-      <FormControl variant="filled" margin="normal" fullWidth>
-        <InputLabel>{t('reportDevice')}</InputLabel>
-        <Select value={deviceId} onChange={(e) => setDeviceId(e.target.value)}>
-          {devices.map((device) => (
-            <MenuItem value={device.id}>{device.name}</MenuItem>
-          ))}
-        </Select>
-      </FormControl>
-      <FormControl variant="filled" margin="normal" fullWidth>
-        <InputLabel>{t('reportPeriod')}</InputLabel>
-        <Select value={period} onChange={(e) => setPeriod(e.target.value)}>
-          <MenuItem value="today">{t('reportToday')}</MenuItem>
-          <MenuItem value="yesterday">{t('reportYesterday')}</MenuItem>
-          <MenuItem value="thisWeek">{t('reportThisWeek')}</MenuItem>
-          <MenuItem value="previousWeek">{t('reportPreviousWeek')}</MenuItem>
-          <MenuItem value="thisMonth">{t('reportThisMonth')}</MenuItem>
-          <MenuItem value="previousMonth">{t('reportPreviousMonth')}</MenuItem>
-          <MenuItem value="custom">{t('reportCustom')}</MenuItem>
-        </Select>
-      </FormControl>
-      {period === 'custom' && (
+    <Grid container spacing={3}>
```

Comment:
I have tried with margins and padding but have not satisfied with my implementations. Please suggest.

Reply:
Material-ui allows targeting of children elements when creating styles with makeStyles. So you can do something like this 
```
const useStyles = makeStyles({
  gridContainer: {
    '&>*': {
      paddingTop: theme.spacing(1.5),
      paddingBottom: theme.spacing(1.5)
  }
});
```
This will allow you to target the padding of the immediate children